test/capitalize.test.js

```diff
@@ -0,0 +1,35 @@
+import capitalize from "../src/capitalize.js";
+
+describe('capitalize', function() {
+    it('should turn the first letter of a lowercase string to uppercase', function() {
+	expect(capitalize("example")).toBe("Example");
+    });
```

Comment:
```suggestion
    it('should turn the first letter of a lowercase string to uppercase', function() {
	    expect(capitalize("example")).toBe("Example");
    });
```
The indentation should be like above.